scipy/signal/_fir_filter_design.py

```diff
@@ -1280,3 +1281,123 @@ def minimum_phase(h: np.ndarray,
         h_minimum = h_temp.real
     n_out = (n_half + len(h) % 2) if half else len(h)
     return h_minimum[:n_out]
+
+
+def fwind1(hsize, window, fc=None, fs=2, circular=False):
```

Comment:
We should make some of these required keywords as per https://docs.scipy.org/doc/scipy/dev/missing-bits.html#required-keyword-names
Maybe:
```suggestion
def fwind1(hsize, window, *, fc=None, fs=2, circular=False):
```